skimage/feature/blob.py

```diff
@@ -24,50 +23,58 @@ def _blob_overlap(blob1, blob2):
 
     Parameters
     ----------
-    blob1 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
-    blob2 : sequence
-        A sequence of ``(y,x,sigma)``, where ``x,y`` are coordinates of blob
-        and sigma is the standard deviation of the Gaussian kernel which
-        detected the blob.
+    blob1 : sequence of arrays
+        A sequence of ``(row, col, sigma)`` or ``(pln, row, col, sigma)``,
+        where ``row, col`` (or ``(pln, row, col, sigma)``) are coordinates
```

Comment:
Not sure I understand, you need to have an extra dimension for the computation of ``d = sqrt(np.sum((blob1[:-1] - blob2[:-1])**2))``